modules/installation-azure-regions.adoc

```diff
@@ -5,6 +5,32 @@
 [id="installation-azure-regions_{context}"]
 = Supported Azure regions
 
-The list of available Microsoft Azure regions is dynamically generated.
+The installation program dynamically generates the list of available Microsoft Azure regions based on your subscription. The following Azure regions were tested in {product-title} version 4.2:
```

Comment:
`The following Azure regions were tested in {product-title} version 4.2`

I would change that to:

The following Azure regions were tested to be correct in {product-title} version 4.2 GA.